extension-requests/script.js

```diff
@@ -372,141 +340,143 @@ async function createExtensionCard(data) {
     attributes: {
       class: 'external-link',
     },
   });
-
   const taskTitle = createElement({
     type: 'span',
     attributes: { class: 'task-title' },
     innerText: 'Task: ',
   });
-
   taskTitle.appendChild(statusSiteLink);
   taskDetailsContainer.appendChild(taskTitle);
   const detailsLine = createElement({
     type: 'span',
     attributes: { class: 'details-line' },
   });
   detailsContainer.appendChild(detailsLine);
-
   const deadlineContainer = createElement({ type: 'div' });
   taskDetailsContainer.appendChild(deadlineContainer);
-
   const deadlineText = createElement({
     type: 'span',
     attributes: { class: 'card-row-text' },
     innerText: `Deadline${isDeadLineCrossed ? ': ' : ' in: '}`,
   });
   deadlineContainer.appendChild(deadlineText);
-
   const deadlineValue = createElement({
     type: 'span',
     innerText: `${deadlineDays}`,
     attributes: { class: 'tooltip-container' },
   });
   deadlineContainer.appendChild(deadlineValue);
-
   const deadlineTooltip = createElement({
     type: 'span',
     attributes: { class: 'tooltip' },
     innerText: `${fullDateString(secondsToMilliSeconds(data.oldEndsOn))}`,
   });
-
   deadlineValue.appendChild(deadlineTooltip);
-
   const taskStatusContainer = createElement({ type: 'div' });
   taskDetailsContainer.appendChild(taskStatusContainer);
-
   const taskStatusText = createElement({
     type: 'span',
     attributes: { class: 'card-row-text' },
     innerText: 'Task status:',
   });
   taskStatusContainer.appendChild(taskStatusText);
-
   const taskStatusValue = createElement({
     type: 'span',
   });
   taskStatusContainer.appendChild(taskStatusValue);
-
   const datesContainer = createElement({
     type: 'div',
     attributes: { class: 'dates-container' },
   });
   summaryContainer.appendChild(datesContainer);
-
   const datesDetailsContainer = createElement({
     type: 'div',
     attributes: { class: 'details-container' },
   });
   datesContainer.appendChild(datesDetailsContainer);
-
   const extensionDetailsHeading = createElement({
     type: 'span',
     attributes: { class: 'details-heading' },
     innerText: 'Extension Details',
   });
   datesDetailsContainer.appendChild(extensionDetailsHeading);
-
   const extensionDetailsLine = createElement({
     type: 'span',
     attributes: { class: 'details-line' },
   });
   datesDetailsContainer.appendChild(extensionDetailsLine);
 
+  //Added it for displaying extension request number
+  if (params.get('dev') === 'true') {
+    const extensionRequestNumberContainer = createElement({ type: 'div' });
+    datesContainer.appendChild(extensionRequestNumberContainer);
+
+    const extensionRequestNumber = createElement({
+      type: 'span',
+      attributes: { class: 'card-row-text extension-request-number' },
+      innerText: 'Request No.:  ',
+    });
+    extensionRequestNumberContainer.appendChild(extensionRequestNumber);
+
+    //Since previous extensions does not contain requestNumber field
```

Comment:
Can we add a test case for this case as well, i.e., when the request number is not returned in the API response